gitPractice/1-gitHomework.js

```diff
@@ -0,0 +1,45 @@
+//? practice to undrestand difference between const and let.
+//const and let are variable, on the other hand const is better than let because const variable won't be reasigned however let could be reasigned.
+// In conclusion:
+// "Var" variable can be updated and re-declared.don't need initialized.
+// "let" variable can be updated but not re-declared. don't need initialized.
+// "const" variable neither be update and re-declared. needs initialized.
+
+//?practice git and read the codeyourfuture git syllabus.
+//I read the git from codeYourFuture syllabus and also I watched the https://www.youtube.com/watch?v=BCQHnlnPusY and https://www.youtube.com/watch?v=iv8rSLsi1xo
+// Now I know how I can make a repo and how I can commit and add more commit, eventhough we could do the same in VS, git and gitHub.
+// In conclusion I need more practice to pull and push and make changes on it to clear all the options.
+
+//? git push, git pull, what is stage area in git, what is remote repository in git
+//? What is differece between local repository and remote repo in github.
+//"gitPull": If we want to update our branchs with our team mate, we need to write gitpull in terminal to check what have changed, on the other hand we could use git adn github.
+//"StageArea": is a temporary place to put our files to the up. in source control We need to click on + for each files we feel it could be finished, we could press the +.
+//"gitPush": If we want to send our changes in branch, we can write gitpush in terminal and also we could use git or github to update our branches.
```

Comment:
@RominaBideh  What does git push exactly do? explain this to me

Reply:
git push: for example when I want to send my code to my colleague I have to push my file. Although, there are 2 ways to do that. The first way is: I could write git push in the terminal. The second way is: I could use git to push my file and then pull. besides, I learnt how I can mention someone, Whereas, someone with a similar name was mentioned by my bad.

Reply:
@RominaBideh  your explanation is some how correct but you still can not explain to me what the git push is, learn more you tube videos about git push and explain to me as a developer what is git push